articles/supply-chain/master-planning/continue-using-deprecated-planning.md

```diff
@@ -16,8 +16,15 @@ ms.custom: bap-template
 
 [!include [banner](../../includes/banner.md)]
 
-Starting with Supply Chain Management version 10.0.32, it's possible allow some companies (legal entities) to run Planning Optimization while others continue to use the [deprecated master planning engine](deprecated-master-planning-overview.md) until they are ready to be migrated, but you must obtain a special exception from Microsoft to do so (see also [Migration to Planning Optimization for master planning](new-master-planning-engine.md)).
```

Comment:
do we need to mention anything about the exception process?
is it needed it in order to be able to continue using deprecated engine?